doc/html/python.html

```diff
@@ -3161,21 +3107,16 @@ <H3>
       <TD>Removes any underlying math table from the font.</TD>
     </TR>
   </TABLE>
-  <P>
-  <TABLE id="type" border=1>
-    <CAPTION>
-      <A name="Font">font</A>
-    </CAPTION>
-    <TR>
-      <TH colspan=3>Description</TH>
-    </TR>
-    <TR>
-      <TD colspan=3>The font type refers to a fontforge Font object. It generally
-	contains a list of glyphs, an encoding to order those glyphs, a fontname,
-	a list of GPOS/GSUB lookups and many other things.
-	<P>
-	This type may not be pickled.</TD>
-    </TR>
+
+<H2 id="Font">font</H2>
+
+The font type refers to a fontforge Font object. It generally
+contains a list of glyphs, an encoding to order those glyphs, a fontname,
+a list of GPOS/GSUB lookups and many other things.
+<P>
```

Comment:
What's with these `<p>` tags? I don't see the corresponding `</p`> and I don't think you usually nest tables inside of paragraph blocks.

Reply:
Oh, you don't actually have to close p tags. Weird

Reply:
You do in xhtml but not normal old html. Same with dt and dd.